.hemtt/project.toml

```diff
@@ -3,21 +3,38 @@ name = "CBA_A3"
 prefix = "cba"
 author = "CBATeam"
 mainprefix = "x"
-files = [
-    "mod.cpp",
-    "README.md",
-    "LICENSE.md",
-    "logo_cba_ca.paa",
-    "meta.cpp",
-    "userconfig"
-]
 
-version = "3.15.1"
 
 modname = "{{name}}"
 key_name = "{{prefix}}_{{version}}"
 authority = "{{prefix}}_{{version}}-{{git \"id 8\"}}"
 
+
+
+[asc]
+enabled = true
+exclude = [
+        "\\initsettings.sqf",
+        "\\initkeybinds.sqf",
+        "\\xeh_prep.sqf",
+        "\\backwards_comp.sqf",
+        "settings\\gui_createcategory.sqf",
+        "diagnostic\\fnc_isdebugconsoleallowed.sqf",
```

Comment:
even though it's in `exclude` it still hard locks if the included file is missing
```
error: Preprocessor error: `#include` target `[Token { symbol: Escape, source: Position { start: (317, (8, 15)), end: (318, (8, 16)), path: "/addons/diagnostic/fnc_isDebugConsoleAllowed.sqf"
```

Reply:
Fixed.